### PR TITLE
[#91802936] New mongo cluster not coming up good

### PR DESCRIPTION
**Pull request by mreeves1**
- Pin mongo gem to 1.x.x
  A new 2.x gem came up and fouled things up
  ref: https://rubygems.org/gems/mongo/versions
- The gem version I picked is what is currently on prod at:
  /opt/chef/embedded/lib/ruby/gems/1.9.1/gems
- Modified some metadata to keep cookbooks pinned to similar versions to kabam
- Bumped version

### CODE REVIEW

**Comment by coltzhao** [comment]
LGTM.

**Comment by denizipek** [comment]
lgtm

:)

**Comment by mreeves1** [comment]
@denizipek lol! We miss you man. Hope all is well.

File: recipes/mongo_gem.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # install the mongo ruby gem at compile time to make it globally available
 if(Gem.const_defined?("Version") and Gem::Version.new(Chef::VERSION) < Gem::Version.new('10.12.0'))
   gem_package 'mongo' do
+    version '1.12.1'
     action :nothing
   end.run_action(:install)
   Gem.clear_paths
 else
   chef_gem 'mongo' do
+    version '1.12.1'
```
Comment (mattchukabam) [inline recipes/mongo_gem.rb]: Just a comment: make this an attribute?

**Comment by mattchukabam** [comment]
Adding a test wouldn't hurt (wouldn't help much either).

:shipit: !

File: recipes/mongo_gem.rb (hunk 1 above)
Comment (mreeves1) [inline recipes/mongo_gem.rb]: Nah. The issue is that 2.x is clearly not compatible with the rest of the code. So I don't want folks twiddling with it.

**Comment by mreeves1** [comment]
- Pin mongo gem to 1.x.x 
  A new 2.x gem came up and fouled things up 
  ref: https://rubygems.org/gems/mongo/versions
- The gem version I picked is what is currently on prod at: /opt/chef/embedded/lib/ruby/gems/1.9.1/gems
- Modified some metadata to keep cookbooks pinned to similar versions to kabam
- Bumped version

**Pull request merged**